ROADMAP.md

```diff
@@ -35,34 +35,64 @@ issue, in the Slack channel, or in person at the Moby Summits that happen every
 
 ## 1.1 Runtime improvements
 
-We introduced [`runC`](https://runc.io) as a standalone low-level tool for container
-execution in 2015, the first stage in spinning out parts of the Engine into standalone tools.
+Over time we have accumulated a lot of functionaility in the container runtime
+aspect of Moby while also growing in other areas. Much of the container runtime
+pieces are now duplicated work now available in other, lower level components
+such as [containerd](https://containerd.io).
 
-As runC continued evolving, and the OCI specification along with it, we created
-[`containerd`](https://github.com/containerd/containerd), a daemon to control and monitor `runC`.
-In late 2016 this was relaunched as the `containerd` 1.0 track, aiming to provide a common runtime
-for the whole spectrum of container systems, including Kubernetes, with wide community support.
-This change meant that there was an increased scope for `containerd`, including image management
-and storage drivers.
+Moby currnetly only utilizes containerd for basic rutime state management, e.g. starting
+and stopping a container, which is what the pre-containerd 1.0 daemon provided.
+Now that containerd is a full-fledged container runtime which supports full
+container life-cylce management, we would like to start relying more on containerd
+and removing the bits in Moby which are now duplicated. This will neccessitate
+a signficant effort to refactor and even remove large parts of Moby's codebase.
 
-Moby will rely on a long-running `containerd` companion daemon for all container execution
-related operations. This could open the door in the future for Engine restarts without interrupting
-running containers. The switch over to containerd 1.0 is an important goal for the project, and
-will result in a significant simplification of the functions implemented in this repository.
+Tracking issues:
 
-## 1.2 Internal decoupling
+- #38043
+
+## 1.2 Image Builder
+
+Work is ongoing to integrate [BuildKit](https://github.com/moby/buildkit) into
+Moby and replace the "v0" build implementation.
+Buildkit offers better cache management, parallelizable build steps, and better
+extensibility while also keeping builds portable, a cheif tenent of Moby's builder.
```

Comment:
s/cheif/chief/ (unless you were referring to [this urban dictionary use of that word](https://www.urbandictionary.com/define.php?term=Cheif) :joy:)